test/qunit/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <script src="../../dist/js-reporters.js"></script>
+    <link href="../../node_modules/qunitjs/qunit/qunit.css" rel="stylesheet" type="text/css"/>
+    <script src="../../node_modules/qunitjs/qunit/qunit.js"></script>
+    <script src="tests.js"></script>
+
+    <script>
+        var adapter = new JsReporters.QUnitAdapter();
```

Comment:
Can we refer to this as `reporter` or `qunitReporter` (or something more appropriate)? The fact that the reporter is an adapter of the QUnit API and not a native implementation shouldn't matter for its usage. The same applies to the implementation of the `TapReporter` itself.

It should become more obvious once we add a 2nd adapter, at which point we should  implement a `ConsoleReporter` and use it for both frameworks.